### ISSUE TITLE
Bad Instruction in Smart Contract, Losing 0.26 ETH on Ropsten

### ISSUE THREAD

**Issue by mararn1618** [KyberNetwork/smart-contracts#103]
I have just tried to exchange 0.26 ETH for 20 OMG on Ropsten using Metamask, but have lost my funds. I feel this could be pretty significant just before the mainnet launch.

Steps:
* Use Metamask/Ropsten
* Go to https://ropsten.kyber.network/exchange
* Select OMG
* Switch to "From ETH to OMG"
* Switch Advanced on and off
* Enter 20 OMG
* Exchange

Result
* Transaction was mined with a bad instruction on the smart contract
* The 0.26 ETH are lost
* Failed Transaction: https://ropsten.etherscan.io/tx/0xea6a86edd66748199400f086bafb4f1f43b8d4dcfe722a83ff4ce344e13d7f03

If possible I'd like this to be considered in the bug bounty program.

**Comment by yaronvel** [comment]
But the tx was reverted, so you didnt lose your 0.26 eth.

**Comment by mararn1618** [comment]
You are right, the 0.26 ETH were returned. I did confuse this, due to the tx fee of 0.156 lowering my balance significantly. Still worth to investigate, since a high number of failed transactions will leave people frustrated + losing tx fees.
